projects/dvl-fw/src/lib/plugins/activity-log/log-raw-data.ts

```diff
@@ -57,23 +50,18 @@ export class ActivityLogRawData implements RawData {
       activitylog: {
       id: null,
       actionName: get(msg, 'logData.data.type'),
-      fileName: get(msg, 'logData.data.payload.fileName'),
-      fileExtension: get(msg, 'logData.data.payload.fileExtension'),
-      createdUrl: get(msg, 'logData.data.payload.shareUrl'),
-      copiedUrl: get(msg, 'ng logData.data.payload.content'),
-      visualizationId: get(msg, 'logData.data.payload.id'),
-      graphicSymbolId: get(msg, 'logData.data.payload.symbol.recordStream.id'),
-      slot: get(msg, 'logData.data.slot'),
-      visualizationNumber: Number(get(msg, 'logData.data.payload')),
-      date : new Date().toLocaleString()
+      date : new Date().toLocaleString(),
```

Comment:
Please add time : new Date().getTime(). Epoch time is easier to analyze/post-process/sort.

Reply:
@bherr2 Hi Bruce, I have made this change, are you reviewing the complete pull request? do I need another review from Gagan?

Reply:
Gagan should do the review and merge. I'm just lurking 😅

Reply:
ok